source/retryable-reads/retryable-reads.rst

```diff
@@ -314,6 +306,9 @@ The above requirement can be fulfilled in one of two ways:
    there is no guarantee that the lower versioned server can support the
    original command.
 
+.. [1] The first and second commands will be identical unless variations in
```

Comment:
This is one of two or three instances where I relocated footnotes higher up to the end of the section from where they were referenced. The goal was the keep the content closer to its original context and avoid having it down below the changelog, which should ideally be the last section in a spec.